README.md

```diff
@@ -124,15 +126,79 @@ flyway -user=$USER -password="" -locations="filesystem:database/sql/" -url=jdbc:
 
 ### Local Test Network
 
-With a testnet (e.g. [Ganache](https://www.trufflesuite.com/ganache)) running on `localhost:8545` deploy the contracts via:
+In order to run the `e2e` tests you have to have a testnet running locally.
+Due to the RPC calls the services issue `Ganache` is incompatible, so we will use `hardhat`.
+
+1. Install [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)  
+2. Install hardhat with `npm install --save-dev hardhat`  
+3. Create `hardhat.config.js` in the directory you installed `hardhat` in with following content:
+<pre><code>module.exports = {
```

Comment:
ubernit: Why not use <code>```javascript</code> which, AFAIU gets rendered into `<pre><code>` tags with syntax highlighting?

Reply:
Because this code section is part of a list ` ```js ` messed up the rendering of the list. I tried all sorts of combinations of line breaks, `<pre>`, ` ```js `, and `<code>` to not break the list. :'(

Reply:
I think indenting the whole block by `  ` should work (so start and end <code>```</code> as well as the code within the block) - at least in Github flavoured markdown.